gen/container_test.go

```diff
@@ -180,6 +180,9 @@ func TestCollectionsOfPrimitives(t *testing.T) {
 	for _, tt := range tests {
 		assertRoundTrip(t, &tt.p, tt.v, tt.desc)
 		assert.True(t, tt.p.Equals(&tt.p), tt.desc)
+
+		testRoundTripCombos(t, &tt.p, tt.v, tt.desc)
+		assert.True(t, tt.p.Equals(&tt.p), tt.desc)
```

Comment:
Doesn't `testRoundTripCombos` essentially include `assertRoundTrip`? From a brief look, it doesn't look like `assertRoundTrip` checks for anything more than `testRoundTripCombos` with `false,false`.

Reply:
Yeah, we can remove that once all of this integration is done.